src/DynamicObj/DynamicObj.fs

```diff
@@ -98,9 +98,11 @@ type DynamicObj() =
 
     member this.Remove name =
         match this.TryGetPropertyInfo name with
-        | Some pi when pi.IsMutable -> pi.RemoveValue this
+        | Some pi when pi.IsMutable -> 
+            pi.RemoveValue this
+            true
         | Some _ -> failwith $"Cannot remove value for static, immutable property \"{name}\""
-        | None -> ()
+        | None -> false
```

Comment:
Note that this is breaking and needs bumping to 4.x

Reply:
💀